content/k8s/manual-app-deployment/github-setup/_index.md

```diff
@@ -5,7 +5,8 @@ tags:
 - kubernetes
 - github
 prerequisites:
-  hard: []
+  hard:
+  - k8s/manual-app-deployment/setup-ec2
```

Comment:
They actually don't need to have GitHub before having the EC2, they can do this in a different order.

Maybe flag as as soft prereq? Or remove the prereqs?